composer.json

```diff
@@ -104,13 +104,13 @@
         "drupal/autowire_plugin_trait": "^1.0",
         "drupal/azure_mailer": "^2.0",
         "drupal/color_field": "^3.0",
-        "drupal/config_filter": "^2.6",
+        "drupal/config_filter": "^2",
```

Comment:
Please consider not changing the requested version to something looser than what is currently installed. You are effectively saying: I will accept anything that is in the version 2.x range and we seldomly want to go backwards.

As I see it there is nothing in the current version specification that would prevent us from going to `config_filter` 2.7.0 with the current specification.